src/Common/tests/Tests/System/StringTests.cs

```diff
@@ -3987,20 +4050,30 @@ public static void LastIndexOf_TurkishI_TurkishCulture()
                 CultureInfo.CurrentCulture = new CultureInfo("tr-TR");
 
                 string s = "Turkish I \u0131s TROUBL\u0130NG!";
+                ReadOnlySpan<char> span = s.AsSpan();
                 string value = "\u0130";
 
                 Assert.Equal(19, s.LastIndexOf(value));
                 Assert.Equal(19, s.LastIndexOf(value, StringComparison.CurrentCulture));
                 Assert.Equal(19, s.LastIndexOf(value, StringComparison.CurrentCultureIgnoreCase));
                 Assert.Equal(19, s.LastIndexOf(value, StringComparison.Ordinal));
                 Assert.Equal(19, s.IndexOf(value, StringComparison.OrdinalIgnoreCase));
+                Assert.Equal(19, span.LastIndexOf(value.AsSpan()));
+                Assert.Equal(19, span.LastIndexOf(value.AsSpan(), StringComparison.CurrentCulture));
+                Assert.Equal(19, span.LastIndexOf(value.AsSpan(), StringComparison.CurrentCultureIgnoreCase));
+                Assert.Equal(19, span.LastIndexOf(value.AsSpan(), StringComparison.Ordinal));
+                Assert.Equal(19, span.IndexOf(value.AsSpan(), StringComparison.OrdinalIgnoreCase));
```

Comment:
Should this be LastIndexOf, instead of IndexOf? Same with the string-based assert [above](https://github.com/dotnet/corefx/pull/30897/files#diff-91724393075e1a7718d3521655506742R4060)?